backend/app/quiz/schemas.py

```
@@ -0,0 +1,105 @@
+"""
+Pydantic schemas for quiz endpoints.
+"""
+from pydantic import BaseModel, Field, validator
+from datetime import datetime
+from typing import Optional, List
```

Ruff (0.14.1)

`typing.List` is deprecated, use `list` instead

(UP035)